cli/src/filter/resources/jenkins/cli/jenkins-cli-version_pt_BR.properties

```diff
@@ -0,0 +1,23 @@
+# The MIT License
+#
+# Copyright (c) 2004-, Kohsuke Kawaguchi, Sun Microsystems, Inc., and a number
+# of other of contributors.
+# Copyright (c) 2022- Jenkins contributors.
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+version=${project.version}
```

Comment:
Not a localization resource bundle, remove.

Reply:
Will do.
But how can I automatically identify that a Properties is a resource bundle and not localization?
By this example, I could try to find the value beginning with `${` and ending with `}`?

Reply:
Or it would be by the parent directory?

Reply:
This [doc reference](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/ResourceBundle.html) didn't help much.